sst/packages/core/src/util/email.ts

```diff
@@ -0,0 +1,31 @@
+// to replace with fetch when upgraded to Node 18
+import axios from 'axios'
+import { Config } from 'sst/node/config'
+
+const SST_STAGE = process.env.SST_STAGE
+
+export interface Email {
+  recipient: string
+  subject: string
+  body: string
+  from?: string
+  tag?: string
+  // attachments?
+}
+export const sendEmail = async (email: Email) => {
+  const baseURL =
+    SST_STAGE === 'prod'
+      ? 'https://api.postman.gov.sg/v1'
+      : 'https://api-staging.postman.gov.sg/v1'
+  await axios
```

Comment:
Can we use try/catch here instead?

Reply:
hmm there's no difference right? since the only thing that happens in this function is sending the email

Reply:
Materially no diff as there's literally only this function call here so it's rather a nit but I'd rather keep it consistent across the codebase to not use `then.catch` where possible

Reply:
> but I'd rather keep it consistent across the codebase to not use `then.catch` where possible

I'd like to push back against this though. I think `await fnCall().catch()` is a better pattern then `try... catch` the entire thing, because it forces you to think specifically about the I/O boundary where the error would occur.